docs/UserGuide.md

```diff
@@ -245,6 +243,8 @@ Command: `delete id/STUDENT_ID`
 Examples:
 * `delete id/e1234567` deletes the student with the student ID `e1234567`.
 * `delete id/e0589162` deletes the student with the student ID `e0589162`.
+
+_**Note**_: The command to delete both students and events are `delete`. However, `delete` will only delete either a student or an event. I.e. `delete id/STUDENT_ID ev/EVENT_NAME` is not allowed. (Refer to "Deleting an event" for more information.)
```

Comment:
```suggestion
_**Note**_: The command to delete both students and events are `delete`. However, `delete` will only delete either a student or an event. I.e. Including both Student ID and Event prefixes at once: `delete id/STUDENT_ID ev/EVENT_NAME` is not allowed. (Refer to "Deleting an event" for more information.)
```

Reply:
Maybe we should make it more obvious that that command should not contain both prefixes at once